sdk/cognitivelanguage/Azure.AI.Language.Conversations/samples/Sample3_AnalyzeConversationWithLanguage.md

```diff
@@ -29,37 +29,30 @@ Once you have created a client, you can call synchronous or asynchronous methods
 string projectName = "Menu";
 string deploymentName = "production";
 
-var data = new
-{
-    AnalysisInput = new
+AnalyzeConversationInput data =
+    new ConversationalInput(
+        new ConversationAnalysisInput(
+            new TextConversationItem(
+                id: "1",
+                participantId: "1",
+                text: "Enviar un email a Carol acerca de la presentación de mañana")
+            {
+                Language = "es"
```

Comment:
have we considered an enum for supported languages?

Reply:
It will need to be declared in typespec / swagger change to declare it as an enum. That will be a breaking change so I would say nope.